dali/ft/fttransform.cpp

```diff
@@ -640,15 +648,24 @@ void TransferServer::deserializeAction(MemoryBuffer & msg, unsigned action)
         tgtFormat.deserializeExtra(msg, 1);
     }
 
+    unsigned ver = 1; //Using ver=1 to avoid breaking old clients
```

Comment:
minor: no need for temporary 'ver' here, could initialize new 'version', and read into if msg.remaining() , doing away with need for else branch.